src/solver/conv_multipass_wino3x3WrW.cpp

```diff
@@ -381,15 +381,33 @@ bool ConvWinograd3x3MultipassWrW<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>
         if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X3{}) ||
            params.kernel_stride_h == 1)
             return false;
-    if(wino_data_tile == 3 && wino_filter_tile == 4)
-        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X4{}))
-            return false;
-    if(wino_data_tile == 3 && wino_filter_tile == 5)
-        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X5{}))
-            return false;
-    if(wino_data_tile == 3 && wino_filter_tile == 6)
-        if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_MPASS_F3X6{}))
-            return false;
+
+    const std::string name = params.GetStream().GetDeviceName();
```

Comment:
Please add
```C++
#define WORKAROUND_SWDEV_234193 1
````
at the beginning of file and put the workaround code within `#if WORKAROUND_SWDEV_234193... #endif`.